devel/docker/Portfile

```diff
@@ -14,51 +13,30 @@ description         The open-source application container engine
 long_description    Docker is an open source project to pack, ship \
                     and run any application as a lightweight container.
 
-depends_build       port:go
+checksums           rmd160 28d0ad65c73555c779aee7b34daa9032d5d98eac \
+                    sha256 8e77a39b372f4b8ffb9540bc8ae9bcff6516c716a54d68043e2b6b84780b0cf3 \
+                    size   7518166
```

Comment:
The previous alignment of the checksums was correct (columns aligned at multiples of 4 spaces).

Reply:
Fixed, thanks @ryandesign